Sources/EpoxyCollectionView/Models/ItemModel/AnyItemModel.swift

```diff
@@ -68,6 +72,9 @@ extension AnyItemModel: InternalItemModeling {
 
   public func configure(cell: ItemWrapperView, with metadata: ItemCellMetadata) {
     model.configure(cell: cell, with: metadata)
+    if let view = cell.view {
+      setContent?(.init(view: view, metadata: metadata))
+    }
```

Comment:
It would also be nice to support this on `AnySupplementaryItemModel` as well if for symmetry